Formula/openj9.rb

```diff
@@ -0,0 +1,120 @@
+class Openj9 < Formula
+  desc "High performance, scalable, Java virtual machine"
+  homepage "https://www.eclipse.org/openj9/"
+  url "https://github.com/ibmruntimes/openj9-openjdk-jdk16.git",
+    branch:   "v0.25.0-release",
+    revision: "0c11227a21ace48b7c978e29a347822912a6df9c"
+  version "0.25.0"
+  license any_of: [
+    "EPL-2.0",
+    "Apache-2.0",
+    "GPL-2.0-or-later" => { with: "Classpath-exception-2.0" },
```

Comment:
Try
```suggestion
    { "GPL-2.0-or-later" => { with: "Classpath-exception-2.0" } },
    { "GPL-2.0-or-later" => { with: "OpenJDK-assembly-exception-1.0" } },
```

Reply:
Fixed! Thanks, and `clang` supports the `-g` option, but `cc` in shims ignore it, is there any way to re-add it? This formula need this option.

Reply:
@Homebrew/brew ideas?

Reply:
We don't support adding the `-g` option.